ledger-core/v2/virtual/handlers/vdelegatedcallrequest.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2020 Insolar Network Ltd.
```

Comment:
vdelegatedcallrequest? :O 
actually, we have different task for this handler PLAT-313

Reply:
@sexyhamster  ths SM is for sending request, i put in in wrong package. Yes PLAT-311 is for handler